docs/bot/configuration.md

```diff
@@ -43,3 +43,39 @@ These are the keys you can specify:
   "contributors": []
 }
 ```
+
+#### Moving the All Contributors table into a different file
+
+As you know, the default file for the contributors table is `README.md`. But if your project has tons of contributors, it doesn't make sense to include the All Contributors table in the `README.md` file. Instead, you can use a different file to do so.
+
+Here are the steps that may help you in achieving that. For the sake of this tutorial, the different file that we'll use is going to be named `CONTRIBUTORS.md`.
+
+1. Install the bot. See the [Installation](installation) page for instructions.
+2. Add your first contributor. See the [Bot Usage](usage) page for instructions.
+3. Create the `CONTRIBUTORS.md` file.
+4. Copy the code from the Contributors section in the `README.md` file, starting at the `ALL-CONTRIBUTORS-LIST:START` line, and ending at the `ALL-CONTRIBUTORS-LIST:END` line.
+
+   Example:
+   ```html
+   <!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+   <!-- prettier-ignore-start -->
+   <!-- markdownlint-disable -->
+   <table>
+   <tr>
+      <td align="center">...</td>
+   </tr>
+   </table>
+   <!-- markdownlint-restore -->
+   <!-- prettier-ignore-end -->
+   <!-- ALL-CONTRIBUTORS-LIST:END -->
+   ```
+5. Modify the `files` key in the `.all-contributorsrc` JSON file to match your new file name.
+   ```json
+   {
+   "files": [
+      "CONTRIBUTORS.md"
+   ],
+   ...
+   }
+   ```
+6. Now you should be good to go. New contributors will be added to this new file instead of the `README.md` file.
```

Comment:
@hanisirfan I think Step 6 is not necessary. 

@Berkmann18 What do you think?

Reply:
It could be removed, but it's a nice ending note as far as the steps go.